lib/linguist/vendor.yml

```
@@ -60,10 +60,17 @@
 # Bootstrap css and js
 - (^|/)bootstrap([^.]*)\.(js|css|less|scss|styl)$
 - (^|/)custom\.bootstrap([^\s]*)(js|css|less|scss|styl)$
+# Bootstrap with a directory between the files and the main folder like bootstrap/css/bootstrap.css
+- (^|/)bootstrap/(css|js)/.*\.(js|css|less|scss|styl)$
 
 # Font Awesome
 - (^|/)font-awesome\.(css|less|scss|styl)$
 - (^|/)font-awesome/.*\.(css|less|scss|styl)$
+- (^|/)fontawesome\.(css|less|scss|styl)$
+- (^|/)fontawesome/.*\.(css|less|scss|styl)$
```

These two can be consolidated into the two lines above with a single char change - put a `?` after the `-` like this:

```suggestion
- (^|/)font-?awesome/.*\.(css|less|scss|styl)$
```